functions/schema.js

```diff
@@ -15,6 +15,7 @@ const INBOUND_SCHEMA = {
   phoneNumber: 'Phone Number',
   message: 'Message',
   voicemailRecording: 'Voicemail Recording',
+  intakeTime: 'Intake Time',
```

Comment:
not consequential to the code but how are we using this field? it's named... poorly

Reply:
Not sure... Hanna said she wanted it so I put it in there

Reply:
cool- it represents when an inbound was created, not when it became an intake